base/util.jl

```diff
@@ -43,6 +43,8 @@ const disable_text_style = Dict{Symbol,String}(
     :nothing   => "",
 )
 
+const named_styles = Dict{String, NamedTuple{(:color, :properties), Tuple{Union{Symbol,Int}, Dict{Symbol, Bool}}}}()
```

Comment:
Unlike the implementation in https://github.com/JuliaLang/julia/issues/41435#issuecomment-900498232, I've made the values a namedtuple here. This should provide some "future-proofing", if `with_output_color` gains an argument for the background color, see https://github.com/JuliaLang/julia/issues/41435#issuecomment-900425474